source/transfer_list.rst

```diff
@@ -902,5 +902,172 @@ This manifest contains the SPMC attribute node consumed by the SPMD
      - hdr_size
      - Holds a SPMC manifest image in DT format.
 
+
+**Executable entry point information entry layout (XFERLIST_EXEC_EP_INFO)**
+
+This entry type holds one or more `entry_point_info_t` structures.
```

Comment:
I believe it would be better to put each structure in its own TE, rather than packing multiple records into a TE, which we try to avoid. From what I can tell, there is one record per security state, with the 'type' field indicating the state?

Reply:
I tend to agree it would be more flexible for the TE to represent a single entry_point_info but I don't feel strongly about it. However, if we do that, I do feel strongly that the same tag_id should be used for all entries and that there should _not_ be a separate id field, since this would be duplicate information. Parsers can distinguish the entries by parsing all the TEs of this type.
By the way, the type is indicated by the attr field of param_header, not the type field (which tells you that this is an entry_point_info structure). param_header is used for other structures too.

Reply:
Agreed that there shouldn't be a separate id field, since there seems to be a field indicating the type